packages/react-ui/components/Kebab/__tests__/Kebab.test.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import { Kebab } from '../Kebab';
+
+describe('Kebab', () => {
+  it('calls onFocus callback', () => {
+    const onFocus = jest.fn();
+    const kebab = mount(<Kebab onFocus={onFocus} />);
```

Comment:
Метод `mount` возвращает обёртку, поэтому будет корректнее назвать так:
```suggestion
    const wrapper = mount(<Kebab onFocus={onFocus} />);
```